statsd/uds_test.go

```
@@ -0,0 +1,222 @@
+	"os"
+	"testing"
+	"time"
+
+	"golang.org/x/net/nettest"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

golangci-lint (1.62.2)

SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator. 

(staticcheck)